### ISSUE TITLE
Add build instructions for V8

### ISSUE THREAD

**Issue by mihe** [mihe/tsu#37]
Ideally there would be some script(s) bundled with the repo that downloads the currently targeted version of V8 and builds it for some/all platforms. If not, then at least some instructions on how to build a component (DLL) build of V8.

**Comment by gamemake** [comment]
hello, I do some work for this. see: https://github.com/gamemake/tsu/tree/master/Source/ThirdParty/V8/scripts
Now, TSU is possible to run on MacOS.

This work has not been completely done, because v8 compile android needs on linux. In addition v8 7.5 can work in ios.

**Comment by mihe** [comment]
Yeah, I saw your fork the other day @gamemake, it prompted me to make this issue. Nice work. I'll definitely take a closer look at it once I've sorted out some of the bigger remaining issues in [v1.0](https://github.com/mihe/tsu/projects/1).
